<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="Report of BayesPrism Model" />
<meta property="og:type" content="book" />

<meta property="og:description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
<meta name="github-repo" content="rstudio/bookdown-demo" />

<meta name="author" content="Xiangya Wang" />

<meta name="date" content="2024-06-22" />


<meta name="description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>">

<title>Report of BayesPrism Model</title>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libs/navigation-1.1/tabsets.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
/* show arrow before summary tag as in bootstrap
TODO: remove if bootstrap in updated in html_document (rmarkdown#1485) */
details > summary {
  display: list-item;
  cursor: pointer;
}
</style>
</head>

<body>

<div class="container-fluid main-container">


<div class="row">
<div class="col-sm-12">
<div id="TOC">
<ul>
<li><a href="index.html#brief-introduction-to-bayesprism" id="toc-brief-introduction-to-bayesprism">1. Brief introduction to BayesPrism</a></li>
<li class="has-sub"><a href="an-example-of-using-bayesprism.html#an-example-of-using-bayesprism" id="toc-an-example-of-using-bayesprism">2. An example of using BayesPrism</a>
<ul>
<li><a href="an-example-of-using-bayesprism.html#the-high-grade-glioma-dataset" id="toc-the-high-grade-glioma-dataset">2.1 The high-grade glioma dataset</a></li>
<li><a href="an-example-of-using-bayesprism.html#filter-outlier-genes" id="toc-filter-outlier-genes">2.2 Filter outlier genes</a></li>
<li><a href="an-example-of-using-bayesprism.html#construct-prism-object" id="toc-construct-prism-object">2.3 Construct Prism object</a></li>
<li><a href="an-example-of-using-bayesprism.html#run-bayesprism-model" id="toc-run-bayesprism-model">2.4 Run BayesPrism model</a></li>
</ul></li>
<li class="has-sub"><a href="bayesprism-for-our-simulation-data.html#bayesprism-for-our-simulation-data" id="toc-bayesprism-for-our-simulation-data">3. BayesPrism for our simulation data</a>
<ul>
<li><a href="bayesprism-for-our-simulation-data.html#data-from-simulation" id="toc-data-from-simulation">3.1 Data from simulation</a></li>
<li><a href="bayesprism-for-our-simulation-data.html#run-bayesprism-mdoel" id="toc-run-bayesprism-mdoel">3.2 Run BayesPrism mdoel</a></li>
<li><a href="bayesprism-for-our-simulation-data.html#analyzing-the-results" id="toc-analyzing-the-results">3.3 Analyzing the results</a></li>
</ul></li>
</ul>
</div>
</div>
</div>
<div class="row">
<div class="col-sm-12">
<div id="header">
<h1 class="title">Report of BayesPrism Model</h1>
<h4 class="author"><em>Xiangya Wang</em></h4>
<h4 class="date"><em>06/22/2024</em></h4>
</div>
<div id="brief-introduction-to-bayesprism" class="section level2">
<h2>1. Brief introduction to BayesPrism</h2>
<p><strong>Bayes</strong>ian cell <strong>P</strong>roportion <strong>R</strong>econstruction <strong>I</strong>nferred using <strong>S</strong>tatistical <strong>M</strong>arginalization (<strong>BayesPrism</strong>): A Fully Bayesian Inference of Tumor Microenvironment composition and gene expression</p>
<p>BayesPrism consists of the deconvolution module and the embedding learning module.</p>
<p>BayesPrism performs cell type and gene expression deconvolution for bulk RNA-seq using scRNA-seq of samples collected from matched or similar tissue type. It treats the scRNA-seq as prior information, and estimates the joint posterior distribution of cell type fraction and cell type-specific gene expression in each bulk conditional on the reference and each observed bulk.</p>
<p>Since the deconvolution module produces cell type proportions, we mainly concern this part. The following is only about the part of BayesPrism model without the embedding learning.</p>
<p>The input of the model are:</p>
<ul>
<li>bulk samples (# of samples by # of genes)</li>
<li>scRNA_seq (# of cells by # of genes)</li>
<li>cell type labels (# of cell types equal to # of cells in scRNA_seq)</li>
<li>cell state labels (# of subtypes)</li>
</ul>
<p>The output of the model are:</p>
<ul>
<li>Expression matrix of cell types for each sample</li>
<li>Initial cell type fractions for each sample</li>
<li>prosterior cell type fracions for each sample</li>
</ul>
<p>Here this figure is the detailed input and output of the model.
<img src="ff.png" /></p>
</div>
<p style="text-align: center;">
<a href="an-example-of-using-bayesprism.html"><button class="btn btn-default">Next</button></a>
</p>
</div>
</div>


</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

</body>
</html>
